src/test/kotlin/org/rust/ide/docs/RsQuickDocumentationTest.kt

```diff
@@ -23,6 +23,7 @@ class RsQuickDocumentationTest : RsDocumentationProviderTest() {
     fun testGenericStructVariable() = checkDoc()
     fun testTupleDestructuring() = checkDoc()
     fun testConditionalBinding() = checkDoc()
+    fun testStructField() = checkDoc()
```

Comment:
Yes, I like the idea with string literals. For me, it's much more convenient to look at the content of tests in the same file.